.github/actions/comment

```diff
@@ -6,22 +6,41 @@ require_relative "../lib/github"
 stale_label_id = "LA_kwDOEfmk4M8AAAABYVCU-g"
 owner = "community"
 repo = "community"
-only_these_categories = ["Copilot", "Projects and Issues", "Accessibility"]
+only_these_categories = ["Copilot"]
+
+body =<<BODY
+🕒 **Stale Discussion Alert** 🕒
```

Comment:
Settled? Forgotten?

## Automated discussion check-in

Discussion on this item appears to have settled.

* It's possible that's because the question was answered. To save time for our limited contributor base, we ask question authors to mark answers to their discussions so that people who might answer questions don't need to open questions if they're already answered. And similarly, to enable people looking to see answered questions to save their time by not needing to read unanswered questions.

* We understand you're busy, but if you've ruled out the answers that have been provided, providing feedback could help someone provide a better answer.

* If you've found an answer to your own question, we'd really appreciate it if you shared your solution. 

----

I don't like the current proposed text.

"Out of date" might be technically correct, but it's a lousy instruction to a human. Instructions should be based on the querent's use cases, not based on implementation details of the discussion forum.